pandas/io/excel/_base.py

```diff
@@ -345,7 +345,7 @@
 1   string2    2.0
 2  #Comment    3.0
 
-True, False, and NA values, and thousands separators have defaults,
+``True``, ``False``, ``NaN`` values, and thousands of separators have defaults,
```

Comment:
The docs use NA as opposed to NaN in various places because it can refer to pd.NA. Can you revert this change.

Reply:
On it.